speech/setup.py

```diff
@@ -50,14 +50,14 @@
 }
 
 REQUIREMENTS = [
-    'google-cloud-core >= 0.24.0, < 0.25dev',
+    'google-cloud-core >= 0.24.0, < 0.26dev',
     'grpcio >= 1.0.2, < 2.0dev',
-    'gapic-google-cloud-speech-v1beta1 >= 0.15.2, < 0.16dev',
+    'gapic-google-cloud-speech-v1 >= 0.15.3, < 0.16dev',
 ]
 
 setup(
     name='google-cloud-speech',
-    version='0.24.0',
+    version='0.25.0',
```

Comment:
Please hold off on version bumps until a release. This strategy has been problematic (e.g. Spanner's version was incremented in 3 PRs without a release)